README.md

```diff
@@ -36,26 +36,30 @@ This project requires Docker and access to DockerHub.
 The OpenAPI Specification is compatible with [Conjur OSS v1.9+](https://github.com/cyberark/conjur).  
 Clients are generated using [OpenAPI Generator v4.3.1](https://github.com/OpenAPITools/openapi-generator/tree/v4.3.1).
 
-### Use With Dynamic Access Provider (DAP)
+### Use With Conjur Enterprise
 
-The Conjur API endpoints included in this spec are a subset of DAP's endpoints,
-for that reason this API specification (and by extension any generated
-clients/documentation) can be used with DAP.
+An enterprise version of the spec can be generated by calling `./bin/transform --enterprise`
+from within the repository.
 
-The following is a list of endpoints present in DAP but absent from this spec:
+The enterprise version of the spec includes the following exclusive endpoints:
 
 * `/health`
 * `/remote_health`
-* `/configuration/{account}/seed/follower`
 * `/info`
 
+However the `/configuration/{account}/seed/follower` endpoint is still not
+included at this time.
+
 ## Getting Started
 
 If you are new to OpenAPI you may want to visit the
 [understanding the spec](https://github.com/cyberark/conjur-openapi-spec/wiki/Interpreting-The-Spec)
 wiki page, which will give you a brief overview of how to interpret and make changes to the spec file.
 
 To view the Conjur OpenAPI Specification directly, please see the [spec/](./spec/) directory.
+Please note that this version of the spec is not meant to be used itself, you should generate
+either the conjur, or enterprise version for actual use by calling the `bin/transform` script
+with either the `--oss` or `--enterprise` flags.
```

Comment:
this is important, and merits an example. I suggest changing it to:

> To view the source for the Conjur OpenAPI Specification directly, please see the [spec/](./spec/)
directory.
> 
> **Note:** The OpenAPI source includes a [custom property](https://swagger.io/specification/#specification-extensions)
> `x-conjur-settings` to differentiate Enterprise-only endpoints from the rest of the specification.
> In the sections below, we review many common ways that you can use the Conjur OpenAPI definition.
> We recommend using a Conjur OpenAPI definition from an [official release](https://github.com/cyberark/conjur-openapi-spec/releases)
> that's compatible with your specific Conjur server version. If you're interested in building from
> source, you can run the following script to parse the custom `x-conjur-settings` parameters and
> generate an OpenAPI spec for the specific edition of Conjur that you're using:
```
./bin/transform --oss # Creates a spec definition for Conjur OSS in out/oss/
./bin/transform --enterprise # Creates a spec definition for Conjur Enterprise in out/enterprise/
```